docs/changelog.rst

```diff
@@ -3,6 +3,14 @@
 Changelog
 =========
 
+Version 4.0.1
+-------------
+
+* `PR-182 <https://github.com/confluentinc/kafka-connect-elasticsearch/pull/182>`_ - Add 2.x to supported versions
```

Comment:
Not immediately obvious this refers to ES version. Should we add this here? (as we do below?)

Reply:
done.